### DISCUSSION TITLE
v0.9.1

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#408] [Announcements]
1. 添加 Anime4K_Upscale_VL、Anime4K_Upscale_UL、Anime4K_Upscale_Denoise_VL、Anime4K_Upscale_Denoise_UL #403 #406 （感谢 @kato-megumi）
2. 添加 ImageAdjustment，用以调节亮度、对比度等杂项参数 #391 
3. 更新俄语翻译 #388 （感谢 @0x4E69676874466F78 ）
4. 更新 SSimDownscaler
5. 优化部分效果的性能
    * Bicubic 的性能得到大幅提高。CatmullRom 已无明显性能优势，v0.10 将删除此效果
6. 错误修复

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.9.1'>v0.9.1</a>.</em>

**Comment by Crachlow** [comment]
Good job. Thanks

**Comment by V037** [comment]
where is amd super resolution 2.0?

**Comment by hooke007** [comment]
测试了下最新的dotnet v6.0.8 能正常运行

**Reply by Blinue** [reply]
下个版本不再使用 .NET，部署可以简化了

**Comment by OrientxPearL** [comment]
would love to see LS1 added in the future

**Reply by Blinue** [reply]
If it's open source, I'd love to port it

**Comment by LOuroboros** [comment]
> Bicubic 的性能得到大幅提高。CatmullRom 已无明显性能优势，v0.10 将删除此效果

> Bicubic's performance has been significantly improved. CatmullRom no longer has a significant performance advantage, and v0.10 will remove this effect.

Is it really needed to remove the effect? Can't it just be kept for the people who to use it over Bicubic for X or Y reason? 😕

**Reply by hooke007** [reply]
CatmullRom == Bicubic(B=0 C=0.5). To see wiki for more info.
Most traditional algorithms are just the variants of bicubic&spline. And we already support to modify the bicubic's sub-param. There is no need to create the dup effects.

**Reply by LOuroboros** [reply]
> CatmullRom == Bicubic(B=0 C=0.5). To see wiki for more info.

Ah, is it *exactly* the same? Nevermind my question then, thanks 👍

**Reply by hooke007** [reply]
Someone liked `B=C=1/3` and then re-named bicubic to his name `mitchell`. So did some of those variants.
